### PR TITLE
dockerize CromIAM

### PR DESCRIPTION
**Pull request by jsotobroad**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -2,4 +2,6 @@ addSbtPlugin("io.spray" % "sbt-revolver" % "0.8.0")
 
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.3")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.2.0-M7")
+addSbtPlugin("se.marcuslonnberg" % "sbt-docker" % "1.4.1")
```
Comment (cjllanwarne) [inline project/plugins.sbt:5]: Just wanted to double check that this is an industry standard repo, not just "some guy's project"?

**Comment by cjllanwarne** [comment]
Looks good to me! 👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/CromIAM/pull-request/10/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/CromIAM)

File: project/plugins.sbt (hunk 1 above)
Comment (geoffjentry) [reply project/plugins.sbt:5]: regardless, we use it in cromwell ;)

edit: Also, the phrase is "some dude's repo" :)

Comment (jsotobroad) [reply project/plugins.sbt:5]: 👍  to both of Jeff's comments

**Comment by ruchim** [comment]
:100: :+1: 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/CromIAM/pull-request/10/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/CromIAM)

**Pull request merged**